crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/MaatCalculateContributionService.java

```diff
@@ -261,18 +243,6 @@ public boolean isCreateContributionRequired(final CalculateContributionDTO calcu
                 || contributionService.isCds15WorkAround(repOrderDTO)));
     }
 
-    public boolean isEarlyTransferRequired(final CalculateContributionDTO calculateContributionDTO,
-                                           final ApiMaatCalculateContributionResponse response,
-                                           final Contribution latestSentContribution) {
-        return ((response.getMonthlyContributions().compareTo(latestSentContribution.getMonthlyContributions()) != 0
-                || response.getUpfrontContributions().compareTo(latestSentContribution.getUpfrontContributions()) != 0
-                || (latestSentContribution.getEffectiveDate() != null && !response.getEffectiveDate().toLocalDate().equals(latestSentContribution.getEffectiveDate())
-                && BigDecimal.ZERO.compareTo(response.getMonthlyContributions()) < 0)
-                || contributionService.hasCCOutcomeChanged(calculateContributionDTO.getRepId()))
-                && (calculateContributionDTO.getMagCourtOutcome() != null && earlyTransferMagOutcomes.contains(calculateContributionDTO.getMagCourtOutcome())))
-                || contributionService.hasContributionBeenSent(calculateContributionDTO.getRepId());
-    }
-
```

Comment:
It looks like from the logic that because there are no latest sent contributions then this logic won't really return true but does that mean that we no longer need to do early transfers at all? I guess not but something doesnt quite feel right about it...what do you think?